### PR TITLE
Create Enumerable

### PR DESCRIPTION
**Pull request by Mutalenic**
In this project, we have created a class, MyList, and a module MyEnumerable. The module MyEnumerable implements a subset of the functionality of Enumerable.
We made sure that:
- there are no linter errors.
- we used the correct GitHub Flow.
- we documented our work in a professional way.

## Built with

- Ruby

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+This project contain class MyList and a module MyEnumerable. The module MyEnumerable is implementing a subset of the functionality of [Enumerable.](https://ruby-doc.org/core-3.0.0/Enumerable.html)
+
+## Built with
+
+- Ruby
+
+## Getting Started
+
+To start using this software you need to have Ruby installed in your machine
+
```
Comment (TedLivist) [inline README.md]: - [ ] For proper documentation, kindly include specific instructions on how to set up and run the project locally

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (TedLivist) [inline README.md:70]: - [ ] Your MIT link does not work. Kindly fix it. You can create the MIT file and change your URL to match its location, just as we have on the [Microverse template](https://github.com/microverseinc/readme-template).

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Mutalenic and @Ismailco,

Good job so far!
✔️ Implementation of modules
✔️ Implementation of enumerables
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**